README.md

```diff
@@ -694,3 +694,17 @@ require'nvim-treesitter.configs'.setup {
     --
 }
 ```
+
+#### I experience compilation errors when installing certain parsers
+
+Some parsers do not specify a standard for their compilation and rely on the compiler having a sufficient default.
+Check that your compiler is a recent version and update if necessary.
+If you cannot update the compiler, but the one you have supports the minimum standard for the failing parser, you can force compilation via Makefile by adding in your Lua config:
+
+```lua
+local parser_config = require "nvim-treesitter.parsers".get_parser_configs()
+-- For parsers written in c99 or earlier the following line is sufficient.
+parser_config['parser'].install_info.use_makefile = true
+-- For parsers with parts written in c++ you may need to set the following.
+ parser_config['parser'].install_info.cxx_standard = 'c++11'
```

Comment:
when you have this problem, wouldn't you want to use this for all parsers? This line is also from downgrading from c++14 which is the default for the makefile. 

`-- For parsers with parts written in c++ you may need to set the following if your compiler does not support c++14 yet (which is the default for makefile compilation)` 

also leading whitespace.